app/controllers/foreman_deployments/api/v2/stacks_controller.rb

```diff
@@ -17,6 +17,11 @@ def create
           ForemanDeployments::StackParser.parse(@stack.definition)
           process_response @stack.save
         end
+
+        api :GET, '/stacks/', N_('List saved stacks')
+        def index
+          @stacks = Stack.all
```

Comment:
Correct, I plan to add permissions later but the resource scope can be already in the place. I'll add it.